looker_deployer/commands/deploy_content.py

```diff
@@ -134,6 +138,9 @@ def build_spaces(spaces, sdk):
     id_tracker = ["0"]
 
     for space in spaces:
+        # Gazer replaces slashes in folder names with division slashes (Unicode character 2215), so undo that.
+        space = space.replace("\u2215", "/")
```

Comment:
This is an admittedly kludgy way to do this, and runs the risk of mangling folders which had division slashes in their original names. However, I'm betting the number of cases where folders have normal slashes in their names is vastly greater than cases where folders legitimately have division slashes in their names, and the needs of the many outweigh the needs of the few.

The folder JSON files exported by Gazer do contain the original unmangled folder names, so a more robust approach could be to use those folder JSON files when present, but I didn't see a straightforward way to do that given the existing code patterns and Looker Deployer's feature to deploy to a different target folder.